### ISSUE TITLE
train Chinese data 

### ISSUE THREAD

**Issue by yygg678** [MingjieChen/DYGANVC#8]
Great work!  I train a model in Chinese data, the speaker embedding uses nn.Embedding layer, not use speaker encoder netwotk. In test stage, the similarity of inter-gender is very poor. Any  suggestion?

**Comment by MingjieChen** [comment]
Some points to consider:

1. How much data for each speaker in your dataset. E.g. 5min is not very enough to learn nn.Embedding from scratch.
2. You might need to tune hyperparameter to get balance between the reconstruction loss and adversarial loss.

**Comment by yygg678** [comment]
> Some points to consider:
> 
> 1. How much data for each speaker in your dataset. E.g. 5min is not very enough to learn nn.Embedding from scratch.
> 2. You might need to tune hyperparameter to get balance between the reconstruction loss and adversarial loss.

3 hours of data for each speaker, my reconstruction loss is 5.4.

**Comment by MingjieChen** [comment]
So in this case I think you need to do some hyperparameter tuning, e.g. decrease the lambda in front of the reconstruction loss, then maybe increase d_lr.
